Failed to compile uaatokenrefresher

When running the test via `make test`, firehose-to-syslog/uaatokenrefresher_test fails with:

```bash
Failed to compile uaatokenrefresher:

# github.com/elliottpolk/firehose-to-syslog/uaatokenrefresher_test
uaatokenrefresher/uaa_token_refresher_test.go:12:3: err declared but not used
vet: typecheck failures

Ginkgo ran 8 suites in 10.929731115s
Test Suite Failed
Makefile:6: recipe for target 'test' failed
make: *** [test] Error 1
```

--- a/uaatokenrefresher/uaa_token_refresher_test.go
+++ b/uaatokenrefresher/uaa_token_refresher_test.go
@@ -31,7 +31,8 @@ var _ = Describe("UAATokenRefresher", func() {
 	})
 
 	It("fetches a token from the UAA", func() {
-		authToken, err := authTokenRefresher.RefreshAuthToken()
+		var authToken string
+		authToken, err = authTokenRefresher.RefreshAuthToken()
 		Expect(fakeUAA.Requested()).To(BeTrue())
 		Expect(authToken).To(Equal(fakeToken))
 		Expect(err).ToNot(HaveOccurred())

